tools/xcms_refine/test-data/faahKO_reduce.zip

```diff
@@ -0,0 +1 @@
+../../xcms_test-data/faahKO_reduce.zip
```

Comment:
Can you remove this link if you don't use `faahKO_reduce.zip`?
It's in order to reduce the final archive in the toolshed and so the footprint of the tool within the different Galaxy instance

Reply:
Okay, I will check.

Reply:
The tests fail because the wrapped tool requires the RData for input:
```
Input staging problem: Test input file (faahKO-single.xset.merged.RData) cannot be found.
```

I am not in particular favor of skipping the test but am willing to skip them if efficiency demands it.

Reply:
Oh, I didn't saw this test sorry.
Yes, comment it and remove both faahKO-single.xset.merged.RData and faahKO_reduce.zip
The real test about the possibility to still handle zip file is handle upstream by msnbase and fillchrompeaks.